packages/commerce/src/machines/commerce/index.ts

```diff
@@ -299,7 +296,7 @@ const commerceMachine = createMachine<CommerceMachineContext, CommerceEvent>(
         appliedCoupon: (context, _) => {
           const quantity = get(context, 'quantity')
           if (quantity !== 1) {
-            return null
+            return
```

Comment:
returning `null` was causing much more complicated types downstream because `null` is an object/value so setting a type like `mything?: string` doesn't work (the `?` says it can be `undefined` but doesn't cover `null`) so I went through and eliminated the null from the typing and returns in the machine